util/linuxfw/iptables_runner.go

```diff
@@ -48,8 +57,11 @@ func NewIPTablesRunner(logf logger.Logf) (*iptablesRunner, error) {
 
 	supportsV6, supportsV6NAT := false, false
 	v6err := checkIPv6(logf)
+	ip6terr := checkIP6TableExists()
 	if v6err != nil {
 		logf("disabling tunneled IPv6 due to system IPv6 config: %v", v6err)
+	} else if ip6terr != nil {
+		logf("disabling tunneled IPv6 due to missing ip6tables: %v", ip6terr)
```

Comment:
Yeah can both be error but if ipv6 is not supported we wouldn't need to check ip6tables's existance. I think I can do a single if else since these two errors are not on same level.

Reply:
Well at least we wouldn't need both logged at same time.